### PR TITLE
hometask-2 added

### PR DESCRIPTION
**Pull request by bmitioglov**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+let App = () => {
```
Comment (endiny) [inline src/App.js]: Suggest using `const` because the variable won't be redefined

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    return (
+        <div>Hello World!</div>
+    );
+}
+
+ReactDOM.render(<App />, document.getElementById('app'))
```
Comment (endiny) [inline src/App.js]: Also consider installing ESLint to have single code-style within the whole project. I'd recommend to take a look at Airbnb ruleset

**Comment by endiny** [comment]
Optional:

Please take a look at TypeScript. It introduces _almost_ strong typing for JavaScript and helps to reduce some errors on compilation phase preventing them from getting to runtime

File: src/App.js (hunk 2 above)
Comment (bmitioglov) [reply src/App.js]: added ESLint to the project

File: src/App.js (hunk 1 above)
Comment (bmitioglov) [reply src/App.js]: fixed

**Pull request merged**